code/src/components/Sleep.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { SleepIcon } from './SleepIcon'
+
+export const Sleep = ({ sleepHabit, setSleepHabit }) => {
+  const handleUserHabitChange = (event) => {
+    setSleepHabit(event.target.value)
+    console.log(setSleepHabit)
+  }
+  return (
+    <div className="content">
+      <SleepIcon />
+      <h2>Sleep habits</h2>
+
+      <h3>How many hours do you sleep per night?</h3>
+      <div className="radio-buttons">
```

Comment:
To shorten this code you could do an array with all the options, and then .map through them. However I do think your way of doing it is a very good way to get an understanding of how this type of code works.